tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -27,6 +26,7 @@
                         <option value="">----------</option>
                     </select>
                 </div>
+                <span class="fa pficon-help help-tooltip" data-toggle="tooltip" data-placement="left" title="{% trans 'As a tester and/or manager I want to see a breakdown of tests so that I have an idea how the existing tests for the product are distributed' %}"></span>
```

Comment:
Place this at the top of the form like the rest b/c consistency and b/c it doesn't look nice once you resize the screen.